package.json

```diff
@@ -59,12 +58,11 @@
     "verdaccio-disconnect": "source .circleci/local_publish_helpers.sh && unsetNpmRegistryUrl",
     "verdaccio-start": "source .circleci/local_publish_helpers.sh && startLocalRegistry \"$(pwd)/.circleci/verdaccio.yaml\"",
     "verdaccio-stop": "kill -9 $(lsof -n -t -iTCP:4873 -sTCP:LISTEN) || true",
-    "verify-api-extract:clean": "yarn clean && yarn dev-build && yarn verify-api-extract",
+    "verify-api-extract:clean": "yarn clean && yarn build && yarn verify-api-extract",
     "verify-api-extract": "yarn extract-api && ./scripts/verify-extract-api.sh",
     "verify-commit": "yarn ts-node ./scripts/verify-commit.ts",
     "verify-yarn-lock": "./scripts/verify-yarn-lock.sh",
-    "workflow-results": "yarn ts-node ./scripts/cci-workflow-results.ts",
-    "yarn-use-bash": "yarn config set script-shell /bin/bash"
```

Comment:
yarn no longer supports script shell and is using an internal shell interpreter: https://github.com/yarnpkg/berry/discussions/2589